force-app/main/default/layouts/ServiceContract-Service Contract Layout.layout-meta.xml

```diff
@@ -55,6 +55,10 @@
                 <behavior>Edit</behavior>
                 <field>Type__c</field>
             </layoutItems>
+            <layoutItems>
+                <behavior>Edit</behavior>
+                <field>External_Id__c</field>
```

Comment:
Would it make sense to change it to External Key

Reply:
neh

Reply:
fair enough